esp-hal/src/dma/mod.rs

```diff
@@ -1285,15 +1294,78 @@ pub trait ChannelTypes {
     type P: PeripheralMarker;
     type Tx<'a>: Tx;
     type Rx<'a>: Rx;
+    type Binder: InterruptBinder;
+}
+
+pub trait InterruptBinder {
```

Comment:
This can also be `pub(crate)` too?

Reply:
Ah unfortunately not. It appears in `ChannelTypes` and that needs to stay `pub`